contacto.html

```diff
@@ -43,7 +43,7 @@
         <div class="titulo-contacto">
             <h3>Contacto</h3>
         </div>
-        <form action="" class="form-contacto" id = "formulario">
+        <form action="" class="form-contacto" id ="formulario">
             <div>
                 <label for="">Nombre</label>
```

Comment:
Aquí podrías darle valor al atributo ``for` para que al hacerle clic al texto Nombre le haga "focus" al input.
Quedaría en este caso:
```html
<label for="nombre">Nombre</label>
```